sdcm/log_archive.sh

```diff
@@ -0,0 +1,14 @@
+# $1 - file to archive
+# $2 - size splitter in Bytes
+# $3 - archive name
+wc -lc $1 | awk '{print $1,$2}' |  while read lines filesize
+do
+  parts=$(( ( filesize + $2 - 1 ) / $2 ))
+  lines_part=$(( ( lines + parts - 1 ) / parts ))
+  echo "file will be split into $parts parts, per $lines_part lines"
+  for (( i=1; i<=lines; i+=lines_part))
+  do
+    start_time=$(sed -n "${i}p" $1 | awk '{print $2,$3}' | sed -e 's/t://g;s/ /__/g;s/-/_/g;s/:/_/g;s/,/_/g;')
+    tail -n "+$i" "$1" | head -n "$lines_part"  | gzip > "${start_time}.$3.gz"
```

Comment:
To further minimize disk usage, this script could return commands instead of executing them. Later in python, could compress->send to s3->remove part by part.

But this might be an overkill, so I'm not insisting. Just posting an idea.